src/defaults/queue.js

```diff
@@ -5,8 +5,13 @@ function enqueue(array, item, context) {
 }
 
 function dequeue(array, item, context) {
-  const [, ...rest] = array;
-  return rest;
+  let index = array.indexOf(item.meta.offlineAction);
+  if (index >= 0)
+  {
+    let rest = array.slice();
+    rest.splice(index, 1);
```

Comment:
Maybe better to use `.filter`

Reply:
I'm willing to change to .filter and create a new pull request. Could you please tell me which branch to create a pull request to?

Reply:
No need to create a new PR, I can change the base branch. ✌️